sklearn/multioutput.py

```diff
@@ -399,7 +399,7 @@ class MultiOutputClassifier(ClassifierMixin, _MultiOutputEstimator):
     >>> X, y = make_multilabel_classification(n_classes=3, random_state=0)
     >>> clf = MultiOutputClassifier(KNeighborsClassifier()).fit(X, y)
     >>> clf.predict(X[-2:])
-    array([[1, 1, 0], [1, 1, 1]])
+    array([[1, 1, 1], [1, 1, 1]])
```

Comment:
Does this mean the new implementation gives different results than the previous one?

Reply:
`test_neighbors_metrics` normally test consistency of the three algorithms' results.
Yet, in the specific case of this doc-test, the result is different.

I think the best is to test the new `'brute'` back-end against the current implementation.
I will come up with test for this.